Extension.pm

```diff
@@ -203,6 +204,22 @@ sub buglist_column_joins {
     };
 }
 
+##########################################
+# Additional operations when creating bugs
+##########################################
+
+sub object_end_of_create{
+    my ($self, $args) = @_;
+    my ($obj, $old_obj, $changes) = @$args{qw(object old_object changes)};
```

Comment:
The `object_end_of_create` hook gets slightly different arguments

https://www.bugzilla.org/docs/4.4/en/html/api/Bugzilla/Hook.html#object_end_of_create

Also this hook is called after the object has been saved to database, and I'm not sure if there is anything that would add the bug in the pool after that (bug->set_pool_id does not do that directly)

So I wonder if or how this actually works?

Reply:
I'm sorry, going to change parameters.
Yes, the bug is added to the defined pool as the same as we editing the bug.